tests/estimators/object_detection/test_pytorch_yolo.py

```diff
@@ -173,116 +173,116 @@ def test_loss_gradient(art_warning, get_pytorch_yolo):
-                1.99810020e-05,
-                1.70234252e-05,
-                -1.04990377e-05,
-                -2.20760157e-05,
-                2.89993841e-05,
-                2.43352024e-05,
-                5.40295805e-05,
-                -3.54345357e-05,
-                1.49476218e-05,
-                -1.83201955e-05,
-                -4.52892618e-06,
-                -2.10271646e-05,
-                -1.38741379e-05,
-                1.19676406e-06,
-                1.44154410e-05,
-                -1.16514025e-06,
-                -2.48137949e-05,
-                -4.98828103e-06,
-                1.53667770e-05,
-                3.29377417e-06,
-                2.14245338e-05,
-                1.82093459e-06,
-                2.11810093e-05,
-                1.19740625e-05,
-                1.71546981e-06,
-                -1.24681810e-05,
-                -7.98507535e-05,
-                -3.12026459e-05,
-                1.01383330e-05,
-                -3.08082854e-05,
-                -2.56484091e-05,
-                -6.88045111e-05,
-                1.62506112e-05,
-                -1.15138228e-05,
-                -9.07951107e-06,
-                1.25368360e-05,
-                -1.09746734e-05,
+                0.0049910736,
+                -0.008941505,
+                -0.013645802,
+                0.0060615,
+                0.0021073571,
+                -0.0022195925,
+                -0.006654369,
+                0.010533731,
+                0.0013077373,
+                -0.010422451,
+                -0.00034834983,
+                -0.0040517827,
+                -0.0001514384,
+                -0.031307846,
+                -0.008412821,
+                -0.044170827,
+                0.055609763,
+                0.0220191,
+                -0.019813634,
+                -0.035893522,
+                0.023970673,
+                -0.08727841,
+                0.0411198,
+                0.0072751334,
+                0.01716753,
+                0.0391037,
+                0.020182624,
+                0.021557821,
+                0.011461802,
+                0.0046976856,
+                -0.00304008,
+                -0.010215744,
+                -0.0074639097,
+                -0.020115864,
+                -0.05325762,
+                -0.006238129,
+                -0.006486116,
+                0.09806269,
+                0.03115965,
+                0.066279344,
+                0.05367205,
+                -0.042338565,
+                0.04456845,
+                0.040167376,
+                0.03357561,
+                0.01510548,
+                0.0006220075,
+                -0.027102726,
+                -0.020182101,
+                -0.04347762,
             ]
         )
-        np.testing.assert_array_almost_equal(grads[1, 0, 208, 175:225], expected_gradients2, decimal=2)
+        np.testing.assert_array_almost_equal(grads[1, 0, 208, 175:225], expected_gradients2, decimal=1)
```

Comment:
Is the precision change required?

Reply:
Due to normalizing the inputs, the gradients have now changed in magnitude and are much larger now. Therefore, the decimal precision needs to be decreased.